From 0d8c7694872fd1d158c6e08e57513027abe6493b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Mon, 24 Feb 2020 18:11:15 +0100
Subject: [PATCH 06/37] metadata: Set extension compatible with shell 3.36 only

---
 metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index 22ccfca..c74757a 100644
--- a/metadata.json
+++ b/metadata.json
@@ -1,6 +1,6 @@
 {
 "shell-version": [
-    "3.34"
+    "3.36"
 ],
 "uuid": "dash-to-dock@micxgx.gmail.com",
 "name": "Dash to Dock",
-- 
2.26.0

