From 84add82fd23994e63019e7db498f217ee95d43db Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Thu, 27 Feb 2020 17:24:11 +0100
Subject: [PATCH 10/37] docking: Remove unused value

---
 docking.js | 2 --
 1 file changed, 2 deletions(-)

diff --git a/docking.js b/docking.js
index e0e8c37..d89901a 100644
--- a/docking.js
+++ b/docking.js
@@ -1089,8 +1089,6 @@ var DockedDash = GObject.registerClass({
                 this.remove_style_class_name('extended');
             }
         }
-
-        this._y0 = this.y;
     }
 
     _updateStaticBox() {
-- 
2.26.0

