From 9451d36c84b1887c421c41a5ab3b4bf9114628e8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Tue, 17 Mar 2020 00:43:05 +0100
Subject: [PATCH 31/36] docking: Consider theming when computing the
 sliderContainer space

---
 docking.js | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/docking.js b/docking.js
index da63f6d..2712037 100644
--- a/docking.js
+++ b/docking.js
@@ -79,13 +79,14 @@ var DashSlideContainer = GObject.registerClass({
     }
 
     vfunc_allocate(box, flags) {
+        let contentBox = this.get_theme_node().get_content_box(box);
         this.set_allocation(box, flags);
 
         if (this.child == null)
             return;
 
-        let availWidth = box.x2 - box.x1;
-        let availHeight = box.y2 - box.y1;
+        let availWidth = contentBox.x2 - contentBox.x1;
+        let availHeight = contentBox.y2 - contentBox.y1;
         let [, , natChildWidth, natChildHeight] =
             this.child.get_preferred_size();
 
-- 
2.25.1

